release_notes/49-release-notes.adoc

```diff
@@ -523,4 +524,22 @@ This release also addresses the following security vulnerability:
 
 * Denial of service in CivetWeb (link:https://access.redhat.com/security/cve/cve-2025-9648[CVE-2025-9648])
 
+
+[id="about-this-release-492_{context}"]
+== About release 4.9.2
+
+*Release date* {ga-date-492}
+
+This release provides the following bug fixes:
+
+//ROX-31406
```

Comment:
I don't care which one we choose, but I also have this bug in the 4.8.7 release notes so we should be consistent with the language we use and use the same language so people know it's the same issue. Is there a reason to differ from the Release Note that was included in the ticket? From my read of the ticket and the enclosure, the issue is that the casing is inconsistent between the GUI and gitops and this allowed duplicate policies to be created, not that "duplicate policy categories appeared in the console" as you have here. I think this is not exactly a correct interpretation of the bug. I think you tagged AJ to review so perhaps he can weigh in here.

Reply:
4.8.7 notes: https://github.com/openshift/openshift-docs/pull/103304/files